employee-rostering-frontend/public/assets/styles/react-big-calendar.css

```diff
@@ -0,0 +1,512 @@
+@charset "UTF-8";
```

Comment:
`react-big-calendar.css` is copy-pasted from `react-big-calendar/lib/css/react-big-calendar.css`. I copied pasted it since for some odd reason it behaves differently when included as a stylesheet and directly imported (namely, overrides in CSS files without !important don't work). Adding !important to the overrides appears to work.